.venv/project/ecommerce/templates/accounts/login.html

```diff
@@ -0,0 +1,108 @@
+{% extends 'baseLayout.html' %}
+
+
+{% block title %}Login{% endblock %}
+
+{% block head %} 
+<style>
```

Comment:
Css code is now located under the static/css folder. It can be accessed by add the " {% load static %} " tag at the top of the template and reference using href="{% static 'css/login.css' %}".